services/core/shared/core/compat/sdk_v5/accounts.js

```diff
@@ -110,8 +110,8 @@ const getAccountsFromCore = async (params) => {
 		meta: {},
 	};
 	const response = params.addresses
-		? await coreApi.getAccountsByAddresses(params.addresses)
-		: await coreApi.getAccountByAddress(params.address);
+		? await coreApi.requestWithRetries(coreApi.getAccountsByAddresses, params.addresses)
```

Comment:
Can you please add 
```
const requestApi = coreApi.requestWithRetries;
```
at the beginning of those files?

I would advise to use some shorter name btw, such as `requestWithRetries -> requestRetry`